Cargo.toml

```diff
@@ -3,9 +3,10 @@ resolver = "2"
 members = [ 
     "miden-lib",
     "miden-tx",
+    "miden-remote-provers",
     "objects",
```

Comment:
Not for this PR, but we should probably rename `objects` into `miden-objects` and move these crates under `crates` directory (similar to how we have it in the node and client).

Reply:
I created this issue: https://github.com/0xPolygonMiden/miden-base/issues/1062